source/seedsource/management/commands/calculate_zone_transfers.py

```diff
@@ -27,9 +27,12 @@ class Command(BaseCommand):
     def __init__(self, *args, **kwargs):
         self.transfers_by_source = {}
 
-        return super().__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
 
-    def _get_bands_fn(self, source):
+    def add_arguments(self, parser):
+        parser.add_argument('zone_name', nargs=1, type=str, default='all', help='Accepts zone name or all')
```

Comment:
Even though this has a default, it won't let me run the tool without providing the argument. Preferably I should be able to just run ```manage.py calculate_zone_transfers``` and it would do all zones.